Sources/_StringProcessing/Engine/MEBuiltins.swift

```diff
@@ -30,6 +30,27 @@ extension Processor {
     return true
   }
 
+  mutating func reverseMatchBuiltinCC(
```

Comment:
We have zero-width alternation matches (`a|b|`), are there zero width CC matches?

Reply:
Is it possible for a character class to be zero-width?